salt/modules/cmdmod.py

```diff
@@ -864,6 +869,9 @@ def run(cmd,
       variables and set only those provided in the 'env' argument to this
       function.
 
+    :param str prepend_path: $PATH segment to prepend (trailing ':' not necessary)
+      to $PATH
+
```

Comment:
Can you add a ``.. versionadded:: Oxygen`` at the same indent level as the `:param str prepend_path:` line, followed by a blank line? This will properly mark in the docs when the feature was added. You'd also need to repeat this in the rest of the functions where you added this argument.

Reply:
I assume, looking at how other lines like this have been added, it means something like the following:

```
+    :param str prepend_path: $PATH segment to prepend (trailing ':' not necessary)
+      to $PATH
+
+      .. versionadded:: Oxygen
```

Reply:
Yes, that's correct. :)

Reply:
thanks for the confirmation, @rallytime

Reply:
FYI, I've added https://github.com/saltstack/salt/pull/44300/commits/bbdcaee25028f34f202b1ad5b5b04f77739e01d3